client/influxdb.go

```diff
@@ -624,6 +628,17 @@ func (c *Client) Addr() string {
 	return c.url.String()
 }
 
+// checkPointTypes ensures no unsupported types are submitted to influxdb, returning error if they are found.
+func checkPointTypes(p Point) error {
+	for _, v := range p.Fields {
+		switch v.(type) {
+		case uint64:
```

Comment:
this is super old now, if you guys are keen on having this merged (and want to change to specific allow list) let me know and i'll push an update.